## The immune cell atlas of human neuroblastoma
Interactive views of Conos results:
The following links provide interactive views to the Conos joint alignment results shown in the manuscript: (Please use Firefox or Chrome) 

Tutorial video:

[![Watch the video](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/example.png)](https://www.youtube.com/watch?v=12k0vl9wXXQ)


- Joint analysis of all samples [view :mag:](http://pklab.med.harvard.edu/nikolas/pagoda2/frontend/current/pagodaURL/index.html?fileURL=http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/apps//all.all.bin)  [embedding :blue_book:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/all.pdf) [cell annotation & clusters :arrow_down:](http://pklab.med.harvard.edu/shenglin/NB.immune.atlas/data/all.cells.txt)
- Analysis of Myeloid cells [view :mag:](http://pklab.med.harvard.edu/nikolas/pagoda2/frontend/current/pagodaURL/index.html?fileURL=http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/apps/myeloid.all.bin) [embedding :blue_book:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/Myeloid.pdf) [cell annotation & clusters :arrow_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/myeloid.cells.txt)
- Analysis of T cells [view :mag:](http://pklab.med.harvard.edu/nikolas/pagoda2/frontend/current/pagodaURL/index.html?fileURL=http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/apps//Tcell.all.bin) [embedding :blue_book:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/Tcell.pdf) [cell annotation & clusters :arrow_down:](http://pklab.med.harvard.edu/shenglin/NB.immune.atlas/data/T.cells.txt)
- Analysis of B cells [view :mag:](http://pklab.med.harvard.edu/nikolas/pagoda2/frontend/current/pagodaURL/index.html?fileURL=http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/apps//Bcell.all.bin) [embedding :blue_book:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/Bcell.pdf) [cell annotation & clusters :arrow_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/B.cells.txt)
- Analysis of NK cells [view :mag:](http://pklab.med.harvard.edu/nikolas/pagoda2/frontend/current/pagodaURL/index.html?fileURL=http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/apps//NK.all.bin) [embedding :blue_book:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/NK.pdf) [cell annotation & clusters :arrow_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/NK.cells.txt)


## Supplementary data resource:

- Conos object for joint analysis of all samples  [download link :arrow_up_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/all.conos.rds)
- Conos object for Joint analysis of Myeloid cells  [download link :arrow_up_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/myeloid.conos.rds)
- Conos object for Joint analysis of T cells  [download link :arrow_up_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/Tcell.conos.rds)
- Conos object for Joint analysis of B cells  [download link :arrow_up_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/Bcell.conos.rds)
- Conos object for Joint analysis of NK cells  [download link :arrow_up_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/NK.conos.rds)
- Cell annotation  [download link :arrow_up_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/all.cells.csv)
- Signature gene sets  [download link :arrow_up_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/Signature.genes.xlsx)
- Meta data  [download link :arrow_up_down:](http://pklab.med.harvard.edu/shenglin//NB.immune.atlas/data/metaData.xlsx)


## Code availability:
The accession numbers for the raw sequencing data and processed data in this paper are under the accession number: GSE147766.

Code of data analysis is available at https://github.com/shenglinmei/NB.immune.atlas

All other code is available upon request.

